legend-engine-pure-code-compiled-core/src/main/resources/core/pure/binding/functions/functions.pure

```diff
@@ -31,6 +36,12 @@ function meta::external::shared::format::functions::externalize<T>(collection: T
    'Not impemented!';
 }
 
+function meta::external::shared::format::functions::externalize<T>(collection: T[*], binding:Binding[1], graphFetchTree: RootGraphFetchTree<T>[1]):String[1]

```

Comment:
Why still retain externalize function without GFT? I see many TODOs that it will be mandated. Shouldn't we do it now?

Reply:
Plan to remove it in phases so that existing use-cases are not impacted